autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>Please include a link to the homepage of the particular dataset? Some markdown like:</p><p> </p><blockquote>We will use from the [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/index.php) the [Polish companies bankruptcy dataset](https://archive.ics.uci.edu/ml/datasets/Polish+companies+bankruptcy+data)</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
done